examples/llama2/user_script.py

```diff
@@ -297,9 +297,9 @@ def collate_batch(self, batch):
 
             # Pad input data because all model inputs must have same shape
             pad_len = self.pad_max - input_ids.shape[0]
-            input_ids = F.pad(input_ids, (0, pad_len), value=1)
-            attention_mask = F.pad(attention_mask, (0, pad_len), value=0)
-            position_ids = F.pad(position_ids, (0, pad_len), value=0)
+            input_ids = F.pad(input_ids, (0, pad_len), value=1)  # pylint: disable=not-callable
```

Comment:
you can just put # pylint: disable=not-callable above the three lines. Then only one line comment is enough

Reply:
FYI: Be sure to add `# pylint: enable=not-callable` after the lines. Otherwise the rule will be disabled for the rest of the file

Reply:
since the not-callable is false positive at most case, I am considering to disable it globally.